docs/Users_Guide/installation.rst

```diff
@@ -121,6 +121,14 @@ to run.
     - cartopy (0.17.0)
     - matplotlib (3.3.4)
 
+Cartopy, one of the dependencies of CyclonePlotter, attempts to download shapefiles from the internet to complete successfully. So if CyclonePlotter is run on a closed system (i.e. no internet), additional steps need to be taken. First, go to the Cartopy repository and download the tarball for Natural Earth shapefiles. The categories should include
+
```

Comment:
You can replace the sentence "_First, got to the Cartopy repository and download the tarball for Natural Earth shapefiles_" with something like this:

_Download the small scale (1:110 m data) cultural and physical shapefiles from the Natural Earth downloads page: https://www.naturalearthdata.com/downloads/ and save them in a location that is accessible to all users._ 

Or something to that effect...

Reply:
Rolled suggestions into new wording